easypost/beta/referral.py

```diff
@@ -109,6 +109,67 @@ def add_credit_card(
         )
         return convert_to_easypost_object(response)
 
+    @staticmethod
+    def add_payment_method(
+        stripe_customer_id: str,
+        payment_method_reference: str,
+        priority: Optional[str] = "primary",
+    ) -> Dict[str, Any]:
+        """Add a Stripe payment method to your EasyPost account.
+
+        This endpoint uses a user's personal Stripe account. The `stripe_customer_id`
+        and `payment_method_reference` IDs both come from Stripe. By adding these to
+        EasyPost, we will associate your Stripe payment method with either your primary
+        or secondary EasyPost payment method.
+        """
+        requestor = Requestor()
+        wrapped_params = {
+            "payment_method": {
+                "stripe_customer_id": stripe_customer_id,
+                "payment_method_reference": payment_method_reference,
+                "priority": priority,
+            }
+        }
+
+        response, api_key = requestor.request(
+            method=RequestMethod.POST,
```

Comment:
This doesn't use the referral API key

Reply:
Per our discussion, this is setup correctly. You would have a specific `referral` client for this function call.